### ISSUE TITLE
Inconsistent WKWebView sandboxing in MenuBarTrigger breaks EUI-Trigger, somtimes

### ISSUE THREAD

**Issue by kd8drx** [elastic/trigger-eui#1]
Details of the issue are in taniacomputer/MenuBarTrigger#5.

TL;DR WKWebView's caching and sandboxing functions can lead to some weird behavior when loading EUI-Trigger based warnings, based on what order files are called in, what resources they use, and how they're cached.

As a basic workaround, calling `loading.html` or 'spinner.html` as the first HTML page in an MBT script ensure basic EUI CSS is available to subsequent views. However, other resources, like icons or JavaScript often are not - it depends on what was cached when loading the first file.

I've identified a upstream code change that will address the issue, and will be implementing it in this next IT-Eng sprint, as this issue is blocking us from deploying a few notifications to the fleet that are needed.

**Comment by kd8drx** [comment]
For External Visibility: I've added this issue to my current sprint at Elastic, and hope to have a PR upstream with the proposed fix later this week - assuming I have this all sketched out correctly.
